### PR TITLE
Feature/added UI permissions to user endpoint

### PR DESCRIPTION
**Pull request by bartvanb**
This PR adds a few additional details to the user endpoint so that the UI can be more efficient in determining permissions.

### CODE REVIEW

File: vantage6-server/vantage6/server/resource/user.py [hunk 1]
```diff
@@ -495,6 +497,13 @@ def get(self, id):
                 type: integer
               description: User id
               required: true
+            - in: path
+              name: include_ui_permissions
```
Comment (frankcorneliusmartin) [inline vantage6-server/vantage6/server/resource/user.py:501]: I don't like the name `include_ui_permissions` as this implies these permissions are for the UI. But these are actually backend permissions. I would change this to `include_permissions`.

File: vantage6-server/vantage6/server/resource/user.py [hunk 2]
```diff
@@ -495,6 +497,13 @@ def get(self, id):
...
+              schema:
+                type: boolean
+              description: Whether or not to include extra permission info for
+                the user interface. You can only get these for yourself. By
+                default false.
```
Comment (frankcorneliusmartin) [inline vantage6-server/vantage6/server/resource/user.py:506]: Why make this exclusive to your own user? I get that this is specifically written for the UI, but why limited it? If other users have sufficient permissions this should not be a problem?

**Review by frankcorneliusmartin — CHANGES_REQUESTED** [review]

File: vantage6-server/vantage6/server/resource/user.py (hunk 1 above)
Comment (bartvanb) [reply vantage6-server/vantage6/server/resource/user.py:501]: Changed this

**Review by frankcorneliusmartin — APPROVED** [review]
LGTM

**Comment by frankcorneliusmartin** [comment]
Tested using 

```python
client.request('user/1', method='get', params={'include_permissions': True})
client.request('user/1', method='get')
```

Output seems correct. Related issue: #832

**Pull request merged**